components/badge/badge_vibrator.h

```diff
@@ -24,6 +24,16 @@ extern esp_err_t badge_vibrator_init(void);
  */
 extern void badge_vibrator_activate(uint32_t pattern);
 
+/**
+ * Activate the vibrator for the specified number of microseconds.
+ *
+ * Code example:
+ *
+ *   badge_vibrator_activate(200000);
+ *   // vibrator will be on for 200ms.
```

Comment:
used documentation style is doxygen; suggested comment:

    @param length Length in microseconds.